watertap/data/techno_economic/decarbonator.yaml

```diff
@@ -21,11 +21,11 @@ default:
     cost_function_form:
       function f(flow_vol_in): cap_basis_par*(flow_vol_in/flow_basis_par)^cap_exp
       units: $
-  recovery_frac_mass_H2O:
-    value: 0.9999
-    units: dimensionless
-    reference: Survey of High-Recovery and Zero Liquid Discharge Technologies for
-      Water Utilities (2008).
   default_removal_frac_mass_solute:
     value: 0
     units: dimensionless
+  removal_frac_mass_solute:
+    CO2:
+      value: 0.5    #todo: revisit this value later on
```

Comment:
Although it may have little meaning now, and in the absence of data, perhaps you can raise this to something closer to 1 (e.g., 0.999) for the time being, since the intention would be to remove any co2 gas in the stream.

Reply:
Revised the removal mass fraction of carbon dioxide to 0.999.